src/Protocol/Diaspora.php

```diff
@@ -99,6 +99,53 @@ public static function relayList()
 		return $relay;
 	}
 
+	/**
+	 * @brief Return a list of participating contacts for a thread
+	 *
+	 * This is used for the participation feature.
+	 * One of the parameters is a contact array.
+	 * This is done to avoid duplicates.
+	 *
+	 * @param integer $thread   The id of the thread
+	 * @param array   $contacts The previously fetched contacts
+	 *
+	 * @return array of relay servers
+	 */
+	public static function ParticipationsForThread($thread, $contacts)
```

Comment:
Standards: Please use `camelCase` for class method names.

Reply:
Yeah, that's the same like above. At my work we are mostly using this style.